configs/amazon-cloudwatch-publisher-osx.json

```diff
@@ -4,13 +4,7 @@
       "prefix": "osx",
       "command": "system_profiler SPHardwareDataType | grep Serial | awk '{print $NF}'"
     },
-    "authentication": {
-      "accountId": "ACCOUNT_ID",
-      "userPoolId": "USER_POOL_ID",
-      "identityPoolId": "IDENTITY_POOL_ID",
-      "appClientId": "APP_CLIENT_ID",
-      "password": "PASSWORD"
-    },
+    "run_as_user": "_cwpublisher",
```

Comment:
I don't think `run_as_user` was implemented, was it?

Reply:
No, it was not. The "run_as_user" was implemented for Linux only. Will remove.